open_academy/web_services/ws.py

```diff
@@ -0,0 +1,64 @@
+import xmlrpc.client
+
+url = "http://localhost:8069"
+db = "rd-mydb"
+username = "admin"
+password = "admin"
+
+# Authentication
+common = xmlrpc.client.ServerProxy('{}/xmlrpc/2/common'.format(url))
+uid = common.authenticate(db, username, password, {})
+models = xmlrpc.client.ServerProxy('{}/xmlrpc/2/object'.format(url))
+
+# Show sessions and seats
+
+session_info = models.execute_kw(db, uid, password, 'open.academy.session',
+                                 'search_read', [], {'fields': ['name', 'seats']})
+
+for session_and_seats in session_info:
+    print("The session '{}' has {} seats".format(session_and_seats["name"], session_and_seats["seats"]))
+
+print("\n")
+
+# Create a session
+
+courses_info = models.execute_kw(db, uid, password, 'open.academy.course',
+                                 'search_read', [], {'fields': ['name']})
+instructors_info = models.execute_kw(db, uid, password, 'res.partner',
+                                     'search_read', [[['instructor', '=', True]]],
+                                     {'fields': ['name']})
+possible_attendees = models.execute_kw(db, uid, password, 'res.partner',
+                                 'search_read', [], {'fields': ['name']})
```

Comment:
You just need one course and one instructor.
```suggestion
course_id = models.execute_kw(db, uid, password, 'open.academy.course',
                                 'search', [], {'limit': 1})
instructor_id = models.execute_kw(db, uid, password, 'res.partner',
                                     'search', [[['instructor', '=', True]]],
                                     {'limit': 1})
attendee_id = models.execute_kw(db, uid, password, 'res.partner',
                                 'search', [], {'limit': 1})
```